xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/ConfigurableClass.xml

```diff
@@ -181,8 +183,13 @@ $doc.dropPermissions()##
       #set($configurableObjs = [])
       #foreach($configurableObj in $allConfigurableObjs)
         #if($app.getValue('displayInSection', $configurableObj) == $section)
-          ## If this is space admin, then don't display global, if global don't display space.
-          #if($globaladmin == ($app.getValue('configureGlobally', $configurableObj) == 1))
+          ## If this is space admin, then don't display global, if global don't display space. Valid for 
+          ## configureGlobally xproperty.
+          ## visibility xproperty introduced the posibility to configure at both levels wiki&amp;space(global) or 
+          ## at each level individually.
```

Comment:
I would rewrite this comment which is not very clear to me and make a special care to mention that the 'configureGlobally' xproperty is now deprecated in favor of the 'visibility' one.